Sources/IRGen/IULIAEnum.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  IULIAEnum.swift
+//  IRGen
+//
+//  Created by Hails, Daniel J R on 31/07/2018.
+//
+
+import AST
+
+/// Generates code for an enum.
+public struct IULIAEnum {
```

Comment:
Why does a IULIAEnum contain a StructDeclaration? Is this file even used?

Reply:
It shouldn't - It is used because when rawValues are implemented this will generate the array of rawValues.